src/TLuaInterpreter.cpp

```diff
@@ -4264,8 +4255,7 @@ int TLuaInterpreter::getRoomHashByID(lua_State* L)
         return 1;
     }
     lua_pushnil(L);
-    // This is sloppy: it might exist but not be hashed.
-    lua_pushfstring(L, "room %d doesn't exist", id);
+    lua_pushfstring(L, "No hash for room %d.", id);
```

Comment:
As the run-time error message text is likely to be combined with other stuff:
+ no sentence capitalisation please ("**n**o hash ,,," not "**N**o hash...")
+ no sentence termination ("... for room %d" not "... for room %d.")

please... :grinning:

Reply:
Yeah.
Fixed, built, tested, resubmitted.